### PR TITLE
Expanded dict entry view

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

File: src/dictionary/ui/display/WordInformationDisplay.test.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  })
+  it('Word parts are displayed', async () => {
+    word.lemma.map((lemma) => expectToBeVisible(new RegExp(lemma)))
+    expectToBeVisible(new RegExp(word.guideWord))
+    expectToBeVisible(new RegExp(word.meaning))
+
+    function expectToBeVisible(text: string | RegExp): void {
+      screen
+        .getAllByText(text)
+        .map((gettedText) => expect(gettedText).toBeVisible())
```
Comment (jlaasonen) [inline src/dictionary/ui/display/WordInformationDisplay.test.tsx:36]: What's the point of using `map` when you do not use the return value?

File: src/dictionary/ui/display/WordInformationDisplay.tsx [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    href="https://www.harrassowitz-verlag.de/isbn_978-3-447-04264-2.ahtml"
+    title="A Concise Dictionary of Akkadian, Harrassowitz Verlag"
+  >
+    <Row>
+      <Col xs={11} className="mx-auto">
+        <Row className="m-3 py-2 border border-dark">
+          <Col xs="auto">
+            <i className="linkContainer mt-2 fas fa-shopping-cart" />
+          </Col>
+          <Col className="linkContainer my-auto" style={{ fontSize: '0.7em' }}>
```
Comment (jlaasonen) [inline src/dictionary/ui/display/WordInformationDisplay.tsx:26]: Why is the class name `linkContainer`, there is no link inside`?

Use BEM naming conventions.

Do not use inline styles.

File: src/dictionary/ui/display/WordInformationDisplay.tsx [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+          </Col>
+        </Row>
+      </Col>
+    </Row>
+  </ExternalLink>
+)
+
+function WordInformation({ word }: { word: Word }): JSX.Element {
+  const copyableInformation = `+${word.lemma[0]}[${word.guideWord}]${
+    word.pos[0] ? word.pos[0] : ''
```
Comment (jlaasonen) [inline src/dictionary/ui/display/WordInformationDisplay.tsx:40]: You can use `??` the check for null/undefined.

File: src/dictionary/ui/display/WordInformationDisplay.tsx [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+            </Col>
+            <Col>
+              {Boolean(word.pos.length) && (
+                <h5 className="text-secondary">({word.pos.join(', ')})</h5>
+              )}
+            </Col>
+            <Col xs="auto" className="pr-5 mr-5">
+              <div
+                className="border border-dark p-1"
+                style={{ fontSize: '0.5em', color: 'grey' }}
```
Comment (jlaasonen) [inline src/dictionary/ui/display/WordInformationDisplay.tsx:68]: Do not use inline styles.

File: src/dictionary/ui/display/WordInformationDisplay.tsx [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+              >
+                {copyableInformation}
+                <Button
+                  style={{ padding: '3px 5px 0 5px' }}
```
Comment (jlaasonen) [inline src/dictionary/ui/display/WordInformationDisplay.tsx:72]: Do not use inline styles.

File: src/dictionary/ui/display/wordInformationDisplay.css [hunk 6]
```diff
@@ -0,0 +1,6 @@
+.linkContainer:hover {
+  cursor: pointer;
+}
+.markdown {
```
Comment (jlaasonen) [inline src/dictionary/ui/display/wordInformationDisplay.css:4]: What's this for? There is no `markdown` class in the code.

Use BEM naming conventions.

**Review by jlaasonen — CHANGES_REQUESTED** [review]

File: src/dictionary/ui/display/WordInformationDisplay.test.tsx (hunk 1 above)
Comment (yCobanoglu) [reply src/dictionary/ui/display/WordInformationDisplay.test.tsx:36]: i am iterating through the list ? alternative would be using a for loop. `expectToBeVisible `is just void

**Comment by yCobanoglu** [comment]
[Design expanded dictionary entry](https://trello.com/c/fNlwoqVt/31-design-expanded-dictionary-entry)

File: src/dictionary/ui/display/WordInformationDisplay.test.tsx (hunk 1 above)
Comment (jlaasonen) [reply src/dictionary/ui/display/WordInformationDisplay.test.tsx]: `map` is for mapping. `forEach` is for plain iteration.

**Review by jlaasonen — APPROVED** [review]

**Pull request merged**